### PR TITLE
Removed redundant/unused code from codebase

### PR DESCRIPTION
**Pull request by zmhassan**
Before or while filing an issue please feel free to join our [ slack channel](https://join.slack.com/t/mms-awslabs/shared_invite/enQtNDk4MTgzNDc5NzE4LTBkYTAwMjBjMTVmZTdkODRmYTZkNjdjZGYxZDI0ODhiZDdlM2Y0ZGJiZTczMGY3Njc4MmM3OTQ0OWI2ZDMyNGQ) to get in touch with development team, ask questions, find out what's cooking and more!

## Issue #, if available:
#859  
## Description of changes:
Removed unused code or imports from the densenet_pytorch example

## Testing done:

**To run CI tests on your changes refer [README.md](https://github.com/awslabs/mxnet-model-server/blob/master/ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by vdantu** [comment]
@alexwong : Could you review this PR?

File: examples/densenet_pytorch/densenet_service.py [hunk 1]
```diff
@@ -3,21 +3,12 @@
...
 
 import torch
-import time
 import json
-import copy
 import numpy as np
-import PIL
 from PIL import Image
-from collections import OrderedDict
 import torch
```
Comment (alexwong) [inline examples/densenet_pytorch/densenet_service.py]: While we're here, may as well remove this redundant line as well.

**Comment by alexwong** [comment]
Thanks for contributing! Clean code is always good and I think this is an improvement.

File: examples/densenet_pytorch/densenet_service.py (hunk 1 above)
Comment (zmhassan) [reply examples/densenet_pytorch/densenet_service.py]: I removed the other `import torch` higher up line 5

**Comment by zmhassan** [comment]
@alexwong Fixed :)

**Comment by zmhassan** [comment]
Thanks for the speedy review :)

**Comment by alexwong** [comment]
Codebuild is currently failing but it's definitely not due to these changes. Will investigate before merging.

**Comment by vdantu** [comment]
@zmhassan : Thanks for the contribution. Were you able to test that this code works?

**Comment by vdantu** [comment]
@alexwong : Can this be approved and brought in?

**Comment by alexwong** [comment]
Sorry, was dealing w/ some other changes recently. Could you just re-trigger the CI and then we can get the changes in?

**Pull request merged**